site/_posts/2018-01-31-troubleshooting-auth.md

```diff
@@ -514,3 +515,42 @@ Look up groups for a specific user with ManageIQ by doing:
   1. Click *Retrieve*
   1. Near the top of the page a new drop-down should appear populated with group names the specified user is associated with on the IdP
 
+## Example Authentication Config
+---------------------------------------------------------------------
+
+This example, with screenshots, shows one possible authentication configuring.
+
+  + This [Authentication Configuration page screenshot](/assets/images/blog/my_ad_miqldap_auth_config.png) shows a configuration for Active Directory with ***Mode: LDAP(s)***, **User Type** of **Sam Account Name** and **Get User Groups from LDAP** set.
+
+  + This [group add page screenshot](/assets/images/blog/my_ad_miqldap_group_config.png) depicts one example of manually adding a new group by looking up the groups for user **testuser1** in the Identity Provider.
+
+    It is a requirement that at least one group reported by the Identity Provider for each user match a group in the ManageIQ database.
+    Some default groups are provided, others must be manually created.
+     For more information see: [**Adding Groups In ManageIQ**](#adding-groups-in-manageiq)
+
+  + The below examples of the **ldapsearch** command can be used to confirm a user is configured along with group membership associations via the **memberof overlay**.
+
+    When configurng for ***Mode: LDAP(S)*** the [**memberof overlay**](https://technicalnotes.wordpress.com/2014/04/19/openldap-setup-with-memberof-overlay/) is required by default.
+
+  + ldapsearch example using userPrincipalName
+
+    ```
+    $ ldapsearch -x -LLL \
+        -H ldap://MY-AD-server.example.lab.eng.rdu2.redhat.com:389 \
+        -b "DC=MY-AD,DC=example,DC=lab,DC=eng,DC=redhat,DC=com" \
+        -D "my-ad\testuser1" \
```

Comment:
Does the \\ shows up in the blog. Also if this is an actual ldapsearch they would try in the shell, we need to make sure we show \\\\  as the shell escapes those, i.e. -D "my-ad\\\\testuser1"

Reply:
@abellotti Thank you. Yes the `\` do show up in the blog and we don't need to double the backslash because it is in a string quotes. The examples work if as shown.

Reply:
Thanks for checking it out @jvlcek